src/js/game/LevelMVC/LevelView.js

```diff
@@ -546,6 +546,8 @@ module.exports = class LevelView {
     position = [3, 2];
 
     animation = this.playLevelEndAnimation(position, facing, isOnBlock, completionHandler, false);
+    this.game.world.setBounds(0, 0, this.controller.levelModel.planeWidth * 40, this.controller.levelModel.planeHeight * 40);
```

Comment:
should these bounds be set elsewhere? What happens if the player tries to move off the edge of the screen before and after these are set?

Reply:
Good catch — right now they can move off screen during the puzzle.  I will fix!

When we set the bounds for the cut scene animation, the user can no longer control the player so I think it's ok.

Reply:
Note that `this.game.world.setBounds` only constrains the camera.  Sprites don't take the bounds into account when moving.